workflows/demultiplexing/demultiplexing.wdl

```diff
@@ -1,8 +1,8 @@
 version 1.0
 
-import "demuxEM.wdl" as dem
-import "souporcell.wdl" as soc
-import "popscle.wdl" as psc
+import "https://api.firecloud.org/ga4gh/v1/tools/cumulus:demuxEM/versions/6/plain-WDL/descriptor" as dem
```

Comment:
You should use GitHub download URL, not Broad Method Registry URL. For example here, change this line to:
```
import "https://raw.githubusercontent.com/klarman-cell-observatory/cumulus/master/workflows/demultiplexing/demuxEM.wdl" as dem
```